### DISCUSSION TITLE
How to set local host correctly?

### DISCUSSION THREAD

**Discussion by kxxoling** [coollabsio/coolify#2020] [Q&A]
I deployed my instance on a local VM which is `192.168.1.201`, after auto installed the app is served on `http://192.168.1.201:8000/`. I tried to set host to `http://192.168.1.201:8000/` on `/settings` view but it always rewrite to `http://192.168.1.201/`:

![image](https://github.com/coollabsio/coolify/assets/1227139/072e2a32-19d6-45bc-a76a-f60d3d9128c0)

After I set Telegram notification and tested:

<img width="480" alt="image" src="https://github.com/coollabsio/coolify/assets/1227139/fe0e6930-c235-4dc3-943e-836e6c004001">

This URL returns bad gateway error.

**Comment by fcpauldiaz** [comment]
You need to set up a domain and write that in instance domain, the ip is already resolving to your coolify instance

**Comment by peaklabs-dev** [comment]
@kxxoling As fcpauldiaz said, this is for a domain name, the http://192.168.1.201:8000/ is already working without you putting a domain there.
